.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,47 @@
+name: Publish Image on DockerHub
+
+# Action is triggered by pushing version tag to master
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+
+ build:
+    runs-on: ubuntu-latest
+    steps:
+    - 
+      uses: actions/checkout@v2
+      
+    -
+      name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+      
+    -
+      name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+      
+    -
+      name: Login to DockerHub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKER_HUB_USERNAME }}
+        password: ${{ secrets.DOCKER_HUB_TOKEN }}
+        
+    -
+      name: Build and push the image
+      uses: docker/build-push-action@v2
+      with:
+        context: ./focal
+        push: true
+        # Set image name to membraneframeworklabs/docker_membrane:<tag_name>
+        tags: membraneframeworklabs/docker_membrane:${{ github.ref_name }}
```

Comment:
I guess it would be a good practice to push `:latest` tags as well.

Reply:
Fair enough, I was convinced latest would be set automatically 😄